swagger_to_sdk_config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://raw.githubusercontent.com/lmazuel/swagger-to-sdk/master/swagger_to_sdk_config.schema.json",
```

Comment:
@daschult is this temporary or are we thinking of moving the base schema to a tooling branch?

Reply:
This is temporary for now. Eventually I'll publish the updated schema somewhere. I just haven't gotten around to it yet.